### PR TITLE
Update storage samples.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

**Comment by jmdobry** [comment]
Blocked by GoogleCloudPlatform/google-cloud-node#1813

**Comment by jmdobry** [comment]
This is now unblocked.

File: storage/system-test/acl.test.js [hunk 1]
```diff
@@ -16,51 +16,43 @@
...
-      // Ignore error
-      setTimeout(() => {
-        storage.bucket(bucketName).delete(() => {
-          // Ignore error
-          done();
-        });
-      }, 2000);
-    });
+  after(() => {
+    // Try deleting all files twice, just to make sure. Ignore any errors.
```
Comment (ace-n) [inline storage/system-test/acl.test.js]: Nit: this feels sketchy...

Comment (jmdobry) [inline storage/system-test/acl.test.js]: The implementation of `Bucket#deleteFiles` itself is sketchy, and it doesn't always work, causing our tests to randomly fail. Just fighting sketch with sketch.

**Pull request merged**